worldguard-core/src/main/java/com/sk89q/worldguard/commands/region/RegionCommands.java

```diff
@@ -1111,8 +1111,10 @@ public void teleport(CommandContext args, Actor sender) throws CommandException
             }
         }
 
-        player.teleport(teleportLocation,
-                "Teleported you to the region '" + existing.getId() + "'.",
+        String message = existing.getFlag(Flags.TELE_MESSAGE);
+        message = message == null ? Flags.TELE_MESSAGE.getDefault() : message;
+        message = message == null ? "" : message.replace("%id%", existing.getId());
```

Comment:
`getDefault` should return the compile-time default. is this nullcheck necessary?

Reply:
The second nullcheck is actually not necessary.